### ISSUE TITLE
ranger crashed while using dT trash

### ISSUE THREAD

**Issue by cameljava** [ranger/ranger#2664]
#### Runtime Environment

- Operating system and version:  macos 12.4 (Apple M1 Pro)
- Terminal emulator and version:  Iterm2 Build 3.4.15
- Python version: 3.10.5 (main, Jun 23 2022, 17:14:57) [Clang 13.1.6 (clang-1316.0.21.2.5)]
- Ranger version/commit: ranger 1.9.3
- Locale: en_AU.UTF-8

#### Current Behavior
directly in Item2, if using dT to delete highlight file, ranger crash with below error.

#### Expected Behavior
dT will move hightlight file to  mac trash folder.

#### Steps to reproduce
1. 
2. 
3. 

#### Traceback

```
Command not found: `/*'
ranger version: ranger 1.9.3
Python version: 3.10.5 (main, Jun 23 2022, 17:14:57) [Clang 13.1.6 (clang-1316.0.21.2.5)]
Locale: en_AU.UTF-8
Current file: '/Users/kevlee/README.md'

Traceback (most recent call last):
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/core/main.py", line 201, in main
    fm.loop()
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/core/fm.py", line 383, in loop
    ui.handle_input()
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/gui/ui.py", line 266, in handle_input
    self.handle_key(key)
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/gui/ui.py", line 196, in handle_key
    elif not DisplayableContainer.press(self, key):
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/gui/displayable.py", line 275, in press
    focused_obj.press(key)
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/gui/widgets/console.py", line 188, in press
    self.type_key(key)
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/gui/widgets/console.py", line 214, in type_key
    self._answer_question(answer)
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/gui/widgets/console.py", line 197, in _answer_question
    callback(answer)
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/config/commands.py", line 759, in _question_callback
    self.fm.execute_file(files, label='trash')
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/core/actions.py", line 459, in execute_file
    filenames = [f.path for f in files]
  File "/opt/homebrew/Cellar/ranger/1.9.3_2/libexec/ranger/core/actions.py", line 459, in <listcomp>
    filenames = [f.path for f in files]
AttributeError: 'str' object has no attribute 'path'

ranger crashed. Please report this traceback at:
https://github.com/ranger/ranger/issues
```

**Comment by fusate** [comment]
Also experiencing this on Arch linux, using kitty as my terminal emulator, ranger version 1.9.3.  Let me know if I can help troubleshoot anything.

**Comment by markus-bauer** [comment]
I think this issue already exists:
https://github.com/ranger/ranger/issues/2361

If this is the same, then this might be fixed in the master branch (see the discussion there).

Did you search for existing issues before opening this one?

**Comment by harriott** [comment]
Fix reported by @lyhxiangyang in [#2361](https://github.com/ranger/ranger/issues/2361#issuecomment-888886168) worked for my `ranger v1.9.3 on Arch:
```bash
sed -i 's/.path for f in files]/ if isinstance(f, str) else f.path for f in files]/' /usr/lib/python3.10/site-packages/ranger/core/actions.py
```

**Comment by toonn** [comment]
Fixed on master.
